config/env.ts

```diff
@@ -14,7 +14,9 @@ const config: EnvConfig = {
   root: path.join(__dirname, '/..'),
   port: 8080,
   db: process.env.TB_NOTIFICATION_MS_REDIS_DB_STRING,
-  aws_region: process.env.TB_AWS_REGION || 'ap-southeast-1'
+  aws_region: process.env.TB_AWS_REGION || 'ap-southeast-1',
+  sendgrid_key: process.env.SENDGRID_KEY ||
+    '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'
```

Comment:
@abdullahtariq91 can we remove the API keys from here? Its very dangerous to put in git as public open source project 😄